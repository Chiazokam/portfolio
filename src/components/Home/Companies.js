import React from 'react';
import Container from '@material-ui/core/Container';
import Paper from '@material-ui/core/Paper';
import Grid from '@material-ui/core/Grid';

const Companies = () => {
    return (
        <section className='companies-section'>
            <Container fixed>
                <div className='section-title'>Who I Have Worked With...</div>
                <div className='companies-content'>

                        <Grid container spacing={4} className='company-card'>
                            <Grid item md={4} sm={12} xs={12}>
                                <Paper>
                                        <p>Andela</p>
                                </Paper>
                            </Grid>
                            <Grid item md={4} sm={12} xs={12}>
                                <Paper>
                                        <p>Punterclash</p>
                                </Paper>
                            </Grid>
                            <Grid item md={4} sm={12} xs={12}>
                                <Paper>
                                        <p>UBA</p>
                                </Paper>
                            </Grid>
                        </Grid>
                </div>
            </Container>
        </section>
    )
}

export default Companies;
