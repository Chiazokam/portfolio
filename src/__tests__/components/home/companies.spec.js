import React from 'react';
import { shallow } from 'enzyme';
import Companies from '../../../components/Home/Companies';

describe('Companies component', () => {
    it('renders successfully', () => {
    const wrapper = shallow(<Companies />);
    expect(wrapper).toMatchSnapshot();
    })
});
