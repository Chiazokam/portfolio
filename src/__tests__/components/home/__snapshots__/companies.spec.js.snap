// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Companies component renders successfully 1`] = `ShallowWrapper {}`;
